app/services/discovery_engine/evaluation/sample_query_set.rb

```diff
@@ -0,0 +1,62 @@
+module DiscoveryEngine
+  module Evaluation
+    class SampleQuerySet
+      include DiscoveryEngine::Services
+
+      BIGQUERY_DATASET_ID = "automated_evaluation_input".freeze
+      BIGQUERY_TABLE_ID = "clickstream".freeze
+
+      def initialize(month:, year:, sqss_client: sample_query_set_service_client, sqs_client: sample_query_service)
+        @month = month
+        @year = year
+        @sqss_client = sqss_client
+        @sqs_client = sqs_client
+      end
+
+      def create
+        import_data(empty_query_set.name)
+      end
+
+    private
+
+      attr_reader :year, :month, :sqss_client, :sqs_client
+
+      def empty_query_set
```

Comment:
I reckon either the whole GCP query set or at least its name should form part of the object's state - that way we wouldn't have to pass the name separately into the `import_data` method.

Given that objects of this class represent a query set on GCP, would it perhaps make more sense to have a factory method that creates the query set on GCP, and then returns an instance associated with that remote resource?

So you'd go through something like this in the ultimate caller (Rake task):
```ruby
sqs = DiscoveryEngine::Evaluation::SampleQuerySet.create(2025, 5)
sqs.import_from_bigquery(dataset: "automated_evaluation_input", table: "clickstream")
```